### ISSUE TITLE
Set focus to the context menu when it's opened

### ISSUE THREAD

**Issue by Yura13** [isaacplmann/ngx-contextmenu#27]
Set focus to the context menu when it's opened. 
This will make possible to use up and down keys for navigation inside the context menu.

**Comment by isaacplmann** [comment]
Which specific element should the focus be applied to?  The `<ul>`, the first `<li>`, or the `<a>`/`<span>` inside the first `<li>`?

**Comment by Yura13** [comment]
I think the focus should apply to <ul>

**Comment by isaacplmann** [comment]
This has been implemented in 1.2.0

**Comment by Yura13** [comment]
I tested yours changes here(https://isaacplmann.github.io/ngx-contextmenu/). But up and down keys don't work for navigation inside the context menu.

**Comment by isaacplmann** [comment]
Correct.  To navigate with the keyboard, you need to use `tab`, `shift-tab`, `enter` and `esc`.  I spent about a half hour trying to get arrow keys to work and then backed out those changes because it was getting really complicated.

**Comment by Yura13** [comment]
I understand. Will you plan to do this navigation in the future?

**Comment by isaacplmann** [comment]
Probably not.  Unless I figure out how to do it in a simple and maintainable way.  `tab` and `shift-tab` work by using the browser's built in mechanism for changing focus - which I couldn't figure out a way to tap into.

**Comment by Yura13** [comment]
@isaacplmann 
I prepared pull request with navigation inside context menu https://github.com/isaacplmann/ngx-contextmenu/pull/35
In my opinion, after merging this changes autofocus should be removed.

**Comment by isaacplmann** [comment]
Thanks for the contribution!  I'll take a look.

**Comment by isaacplmann** [comment]
PR #35 is incorporated into version 1.3.0.

**Comment by Yura13** [comment]
When using context menu with Bootstrap 4, class 'active' don't apply to item inside a menu. I found that you use bootstrap 3.  The workaround is below:
```
body {
  context-menu-content {
    .dropdown-menu {
      .active {
        .dropdown-item {
          color: #fff;
          text-decoration: none;
          background-color: #0275d8;
        }
      }
    }
  }
}
```

**Comment by isaacplmann** [comment]
It looks like bootstrap 4 expects `.dropdown-menu` to have child `.dropdown-item`s that then have `.active` class applied to them.  I'm missing the `.dropdown-item` class.  I'll add that to the template.

**Comment by Yura13** [comment]
There are used .dropdown-item:active for styling item

**Comment by Yura13** [comment]
Also focus and navigation are async
![image](https://user-images.githubusercontent.com/17030431/27699698-cc2f7b12-5d03-11e7-8dfb-01605a015c0a.png)

**Comment by isaacplmann** [comment]
Yes, I don't plan to try to synchronize tab focus and the keyboard navigation.  This way you can tab through available links just like normal - I'm not hijacking the tab key at all.

**Comment by isaacplmann** [comment]
If you don't like the focus, just turn off autoFocus.

**Comment by isaacplmann** [comment]
This should be fixed in 1.3.1
